src/main/groovy/org/assertj/generator/gradle/internal/tasks/AssertionsGeneratorReport.groovy

```diff
@@ -17,24 +17,25 @@ import org.assertj.assertions.generator.AssertionsEntryPointType
 
 import static com.google.common.collect.Maps.newTreeMap
 import static com.google.common.collect.Sets.newTreeSet
-import static org.apache.commons.lang3.ArrayUtils.isNotEmpty
 import static org.apache.commons.lang3.StringUtils.remove
 import static org.apache.commons.lang3.exception.ExceptionUtils.getStackTrace
 
 class AssertionsGeneratorReport {
```

Comment:
Review this with whitespace checks off, it's _much_ more obvious what I changed.